projects/plugins/jetpack/modules/theme-tools/random-redirect.php

```diff
@@ -38,12 +44,12 @@ function jetpack_matt_random_redirect() {
 	}
 
 	// Acceptable URL formats: /[...]/?random=[post type], /?random, /&random, /&random=1
-	if ( ! isset( $_GET['random'] ) && ! in_array( strtolower( $_SERVER['REQUEST_URI'] ), array( '/&random', '/&random=1' ) ) ) {
+	if ( ! isset( $_GET['random'] ) && ! in_array( strtolower( $_SERVER['REQUEST_URI'] ), array( '/&random', '/&random=1' ), true ) ) { // phpcs:ignore WordPress.Security.NonceVerification.Recommended
```

Comment:
I might just put this on the top of the file

```php
// phpcs:disable WordPress.Security.NonceVerification.Recommended -- No change to the site here, it just redirects.
```
instead of having `phpcs:ignore` on 7 lines.